tools/lib/provision.py

```diff
@@ -247,12 +247,26 @@ def main(options):
         run(["sudo", "service", "redis-server", "restart"])
         run(["sudo", "service", "memcached", "restart"])
     if not options.is_production_travis:
-        # These won't be used anyway
-        run(["scripts/setup/configure-rabbitmq"])
-        # Need to set up Django before using is_template_database_current.
+
+        # Need to set up Django before using is_template_database_current and rabbitmq credentials
         os.environ.setdefault("DJANGO_SETTINGS_MODULE", "zproject.settings")
         import django
         django.setup()
+
+        import pika
+        from django.conf import settings
+        # These won't be used anyway
+        # check if rabbitmq is already configured
```

Comment:
This is another non-sequitur comment waiting to happen, some more months in the future. Perfect example! :-P

In this case, what this comment is trying to do is summarize this chunk of code. I think a better way of expressing that information would be as the name of this code as a separate function.